### ISSUE TITLE
failed to build  "Cannot enable broadcom vulkan driver"

### ISSUE THREAD

**Issue by ghost** [abergmeier/mesa-vulkan-broadcom#2]
```
meson.build:252:2: ERROR:  Problem encountered: Cannot enable broadcom vulkan driver
```

**Comment by tlosm** [comment]
Hi, i have the  same issue here: meson.build:252:2: ERROR:  Problem encountered: Cannot enable broadcom vulkan driver
 ... but i think it become because im building it on x86 debian bullseye.
Please make the source i386 compatible (pls dont insert the architecture control) because for a project i need to build the v3d as native for i386 too ... thank you!!!

**Comment by ghost** [comment]
I think this only works with propietary drivers active (not mesa)

**Comment by tlosm** [comment]
No sorry,
they are Vulkan patch of V3D in mesa. but it can not build on others architectures. In my case i usually build mesa in i386 debian and use the executables in a qemu chroot of my rasperry pi4.

**Comment by tlosm** [comment]
Askmewho ... probably i found how you can build it for the native platform ... do like this 
it start building on my raspian on pi4 

mkdir build
cd build 
meson -Dplatforms=x11,drm,surfaceless -DI-love-half-baked-turnips=true -Dvulkan-drivers=broadcom,freedreno -Dgallium-drivers=v3d --prefix=/usr --libdir=lib/arm-linux-gnueabihf -Dlibunwind=false -Dllvm=false ../mesa-vulkan-broadcom/
ninja

**Comment by tlosm** [comment]
it build on my pi4 this is the result
pi@raspberrypi:~/src/Vulkan $ vulkaninfo 
==========
VULKANINFO
==========

Vulkan Instance Version: 1.1.97



Instance Extensions:
====================
Instance Extensions	count = 13
	VK_EXT_acquire_xlib_display         : extension revision  1
	VK_EXT_debug_report                 : extension revision  9
	VK_EXT_debug_utils                  : extension revision  1
	VK_EXT_direct_mode_display          : extension revision  1
	VK_EXT_display_surface_counter      : extension revision  1
	VK_KHR_display                      : extension revision 23
	VK_KHR_external_memory_capabilities : extension revision  1
	VK_KHR_get_display_properties2      : extension revision  1
	VK_KHR_get_physical_device_properties2: extension revision  1
	VK_KHR_get_surface_capabilities2    : extension revision  1
	VK_KHR_surface                      : extension revision 25
	VK_KHR_xcb_surface                  : extension revision  6
	VK_KHR_xlib_surface                 : extension revision  6
Layers: count = 0
=======
Presentable Surfaces:
=====================
GPU id       : 0 (VideoCore 4.2)
Surface type : VK_KHR_xcb_surface

**Comment by ghost** [comment]
Could you perform any test?? This is too young I guess

**Comment by tlosm** [comment]
yes i test  it with some testing programs found someware ... the vulkan query start and gave a device vulkan capable present like this GPU id : 0 (VideoCore 4.2) ... but after  for all the prg i have only the result of  segmentation failure .. i gdb it and the failur is at 0x0000000 ... right now is not usable ... for what i understand the author here is not interested any more to make it.. plus i subitted a smal patch for fix a compile error but him dint review it but rejected it.
